### PR TITLE
Creating profile for raphink

### PR DESCRIPTION
**Pull request by binford2k**
Hello Raphaël Pinson, and welcome to _Puppet Champions_!

Your first task is to update this PR to match what you'd like your profile to
look like, and then approve it when you're done. Switch to the _Files changed_ tab, then
click the `...` icon at the upper right of this file to access the editor.
[[See GitHub documentation](https://help.github.com/en/articles/approving-a-pull-request-with-required-reviews)]

We use the Jekyll templating system for this site. The first section, between
the dashed lines, is called [_Front Matter_](https://jekyllrb.com/docs/front-matter/). It defines certain variables that we use
to construct your profile. Do not change the keys (the part before the `:`) or the layout.

* The `name` key defines the name displayed on the profiles listing and the headline of your own profile page.
* The `image` key should point to a url of the image you'd like to appear for your profile.
  * If you'd like, you can add it to this repository in the [assets/images/profiles](https://github.com/puppet-community/puppet-community.github.io/tree/master/assets/images/profiles) directory.
  * You'd refer to an image there with a URL of `/assets/images/profiles/{yourname}.jpg`.
* The `bio` text can be anything you want. It will be displayed at the top of your profile
  page, and the first sentence or so will be displayed for _Extraordinary Puppeteers_.

Everything below that Front Matter section is free-form and uses [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/).

After you approve this PR, an admin will merge it and your profile will appear on the site.
If you'd also like to join our fledgling Dev.to organization and post on the behalf of the
Puppet Champions group, then you can do that by navigating to your [settings page](https://dev.to/settings/organization)
and then pasting in the following secret code to join the org:

    c6bc72f90528cf3a0bfade5ea63e09054cb7b247f8c546ddbfb635c07885cb94ba14fe07bbe720d566ce577b3be16d5e4834

You're welcome to come back and update your profile any time you'd like.

### CODE REVIEW

**Comment by binford2k** [comment]
oh hey. This is a public repo, I can't include the secret! 😆

File: _puppeteers/raphink.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: profile
+name: Raphaël Pinson
+image: https://avatars2.githubusercontent.com/u/650430?v=4
```
Comment (raphink) [inline _puppeteers/raphink.md]: Hmmm, everyone seems to use an actual picture as GitHub avatar... where mine is an octocat...

**Comment by alexjfisher** [comment]
@binford2k Could I have one of these dev.io secrets too?

**Comment by raphink** [comment]
@alexjfisher it's the same secret for every one to join the org.

**Comment by alexjfisher** [comment]
`The given organization secret was invalid.`

**Comment by binford2k** [comment]
Heh. I regenerated it when I realized that this was a public repo and I was dumb 😝

**Pull request merged**